opal/util/info_subscriber.c

```diff
@@ -348,6 +349,12 @@ int opal_infosubscribe_subscribe(opal_infosubscriber_t *object, char *key, char
     opal_hash_table_t *table = &object->s_subscriber_table;
     opal_callback_list_item_t *callback_list_item;
 
+    if (strlen(key) > OPAL_MAX_INFO_KEY-strlen("OPAL_INFO_SAVE_PREFIX")) {
```

Comment:
I think your strlen (here and a few lines down) is incorrect -- shouldn't have the quotes.

How about having it abort, but only if it's a debug build?  That should be pretty safe -- normal users don't make debug builds.

Reply:
sounds good, and wow, what an error

Reply:
Is that #if OPAL_ENABLE_DEBUG ?

Reply:
Yep!